main.py

```diff
@@ -866,13 +869,20 @@ def parse_args(self) -> None:
 
         parser.add_argument(
             "--sarif",
-            help="DEPRECATED sarif is currently deprecated, for same functionality as before please use --outPutFormat='sarif'",
+            help="DEPRECATED sarif parameter is currently deprecated, for same functionality as before please use --outPutFormat='sarif'",
             type=bool,
             default=None,
             required=False
         )
 
+        
+
         log(f"Parsing Arguments")
+        # parse help argument
+        help: Namespace = parser.parse_args(["-h", "[--help]"])
```

Comment:
does this overwrite the default --help command?

Reply:
> does this overwrite the default --help command?

Yep, thought we wanted to replace the original help behaviour but now I saw your PR